supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -513,6 +490,19 @@ Always spell out the full product name of the host, and do not capitalize the te
 
 *See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]
 
+// BxMS: General; kept as is
+[discrete]
```

Comment:
@emmurphy1 can you please remove this [discrete] tag in this PR? We no longer use this in our glossary entries (see #365 for details)